src/commands/deploy.ts

```diff
@@ -163,6 +164,9 @@ export async function deploy(context: IActionContext, confirmDeployment: boolean
     const browseWebsite: MessageItem = { title: 'Browse Website' };
     const streamLogs: MessageItem = { title: 'Stream Logs' };
 
+    // referring to this issue https://github.com/Microsoft/vscode-azureappservice/issues/644
+    // waiting 10 seconds is a temporary stop gap and should be changed
+    await delay(10000);
```

Comment:
If we're going to do a delay, I think it should at least happen while the "Deploying..." notification is still showing